pyclang/idf_extension.py

```diff
@@ -1,8 +1,18 @@
 import os.path
 import sys
-
+import shutil
 from pyclang import Runner
 
+def check_esp_clang_tidy():
+    clang_path = shutil.which("clang-tidy")
+    # xtensa is used only till ESP-IDF v5.0
+    if not clang_path or not any(sub in clang_path for sub in ('esp-clang', 'xtensa-esp32-elf-clang')):
+        raise SystemExit(
+            'ERROR: Espressif Clang was not found on the system.\n\n'
+            'For installation instructions, generate and open the documentation with:\n'
+            'idf.py docs --starting-page=api-guides/tools/idf-clang-tidy.html tools/idf-clang-tidy.html'
```

Comment:
I don't think the starting page would be necessary.

Reply:
I've tried the suggested command `idf.py docs --starting-page=api-guides/tools/idf-clang-tidy.html`, which opened the docs on the default landing page - `https://docs.espressif.com/projects/esp-idf/en/latest/esp32/`.

I wasn't able to start it on `https://docs.espressif.com/projects/esp-idf/en/latest/esp32/api-guides/tools/idf-clang-tidy.html` without `--starting-page` option.

But seems to work without the `tools/idf-clang-tidy.html`.

Reply:
Yes, sorry. This is the one which works: `idf.py docs -sp api-guides/tools/idf-clang-tidy.html`

Reply:
Should I use the shorter format instead?